docs/source/examples/tech_tutorial.ipynb

```diff
@@ -0,0 +1,372 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# `Osier` Technology Tutorial\n",
+    "\n",
+    "\n",
+    "Objectives:\n",
+    "\n",
+    "* Learn how to import a pre-defined technology from the ``osier`` technology library.\n",
+    "\n",
```

Comment:
Can you add an explanation of what a technology is, and what its function is within osier?